doc/readme/INSTALL.md

```diff
@@ -291,9 +291,9 @@ Please see any of our Windows py3X [Azure Pipelines](https://dev.azure.com/shogu
 
 1. Install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
 
-2. Open the start menu and run _Anaconda Prompt (Miniconda3)_
+2. Open the start menu, and run _Anaconda Prompt (Miniconda3)_
 
-3. Find the path of the _Native Tools Command Prompt for VS_ relative to your system and Visual studio (x32/x64, 2017/2019). For example for _x64 Native Tools Command Prompt for VS 2019_ it looks so: `%comspec% /k "X:\Path\To\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"`
+3. In the Windows Start menu, find the path of the _Native Tools Command Prompt for VS_ shortcut that is relative to your system architecture and Visual Studio version (x32/x64, 2017/2019/..). For example, for _x64 Native Tools Command Prompt for VS 2019_, it looks so: `%comspec% /k "X:\Path\To\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"`. If it is not enough information, look at [link 1](https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line) and [link 2](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
```

Comment:
`For detailed information check out the following links...`

Reply:
Agree, looks informal